### PR TITLE
Resource: Navigating a large codebase

### PR DESCRIPTION
**Pull request by ramonh**
This also renames the nav link to "resources"

### CODE REVIEW

**Review by deammer — CHANGES_REQUESTED** [review]
@ramonh please hold off on merging, I think we should add a redirect since the `/learn` page has been crawled by search engines

File: gatsby-config.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ module.exports = {
     title: "OSS Port",
   },
   plugins: [
+    "gatsby-plugin-netlify",
```
Comment (deammer) [inline gatsby-config.js:10]: This will generate redirects based on the `static/_redirects` file

**Review by deammer — APPROVED** [review]
Okay good to go!

**Comment by ramonh** [comment]
@deammer Oh this is amazing!! Thank you so much for catching this!

**Pull request merged**